### PR TITLE
Replace SWIFT-Review with SWIFT-Active Screener

### PR DESCRIPTION
**Pull request by lastoel**
The original overview contained SWIFT-Review. However the inclusion criteria we maintain is that the software must employ Active Learning, where SWIFT-Review does not. It turns we mistakenly switched out SWIFT-Review with its AL-counterpart: SWIFT-Active Screener. See [this entry ](https://www.sciome.com/faqs/how-does-swift-active-screener-differ-from-swift-review/)on the Sciome website on what the differences are between the two pieces of software.

This pull request aims to replace all information on SWIFT-Review with the corresponding information on SWIFT-Active Screener. This includes names and links, but also changes in how the software compares in the different feature tables.

### CODE REVIEW

**Comment by lastoel** [comment]
The original overview contained SWIFT-Review. However the inclusion criteria we maintain is that the software must employ Active Learning, where SWIFT-Review does not. It turns we mistakenly switched out SWIFT-Review with its AL-counterpart: SWIFT-Active Screener. See [this entry ](https://www.sciome.com/faqs/how-does-swift-active-screener-differ-from-swift-review/)on the Sciome website on what the differences are between the two pieces of software. 

This pull request aims to replace all information on SWIFT-Review with the corresponding information on SWIFT-Active Screener. This includes names and links, but also changes in how the software compares in the different feature tables.

**Comment by Rensvandeschoot** [comment]
Thank you for making the changes!

**Review by Rensvandeschoot — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ The table below provides an overview of input/output data.
...
 |        [Rayyan](#rayyan)        |    RIS, ENW, BIB, CSV, XML, CIW, NBIB     |             :white_check_mark:(M)             |     RIS, BIB, ENW, CSV      |     :white_check_mark:      |             :x:             |     :white_check_mark:      |
 |  [RobotAnalyst](#robotanalyst)  |                 RIS, NBIB                 | :white_check_mark::grey_question:<sup>5</sup> | :grey_question:<sup>5</sup> |     :white_check_mark:      | :grey_question:<sup>5</sup> |             :x:             |
-|  [SWIFT-Review](#swift-review)  |   RIS, TXT<sup>1</sup>, CSV, XML, NBIB    |             :white_check_mark:(M)             |             TXT             |     :white_check_mark:      |     :white_check_mark:      |             :x:             |
+|  [SWIFT-Active Screener](#swift-activescreener)  |   TXT, RIS, XML, BibTex    |             :white_check_mark:(M)             |             CSV, RIS             |     :white_check_mark:      |     :grey_question:      |             :white_check_mark:             |
```
Comment (gimoAI) [inline README.md:79]: We should add a footnote to [this issue](https://github.com/Rensvandeschoot/software-overview-machine-learning-for-screening-text/issues/40) for the question mark about the re-import. I will add it to the issue (and rename the issue to SWIFT-Active Screener)

File: README.md [hunk 2]
```diff
@@ -119,14 +119,15 @@ The tables below provide an overview of the machine learning properties.
 |      [FASTREAD](#fastread)      | :white_check_mark: |             ≥1R             |
 |        [Rayyan](#rayyan)        | :white_check_mark: |        ≥50 with ≥5R         |
 |  [RobotAnalyst](#robotanalyst)  | :white_check_mark: |             ≥1R             |
-|  [SWIFT-Review](#swift-review)  | :white_check_mark: |        ≥10 with ≥1R         |
+|  [SWIFT-Active Screener](#swift-activescreener)  | :white_check_mark:<sup>2</sup> |        ≥1R+≥1I         |
```
Comment (gimoAI) [inline README.md:122]: > SWIFT-Active Screener can use a training seed to help the software to build the prioritization model more quickly. However, providing a seed is optional and the seed can only be provided before the screening process starts. Currently, Active Screener allows users to upload only relevant documents as a training seed and ideally, these documents would be a ‘representative’ sample of the types of the documents you expect to encounter in the dataset.

So then it should be `≥1R`. If no initial training seed is provided, then:
> Screening proceeds in “batches” of a fixed size, n (here n = 30). After each set of n articles has been reviewed, the model is retrained using all previously screened articles, and the remaining articles are then re-sorted according to this revised model. In this way, the predictions for relevance are updated every 30 documents to incorporate feedback from users as screening continues.

So in that case it would be `≥30 with ≥1R`.

So minimum training data depends on whether the user provides an initial training seed. It can be either `≥1R` or `≥30 with ≥1R`, maybe we should mention these both and explain this with a footnote?

File: README.md [hunk 3]
```diff
@@ -119,14 +119,15 @@ The tables below provide an overview of the machine learning properties.
...
 
 :white_check_mark: Yes/Implemented;
 :x: No/Not implemented;
 :zap: With some effort (add a footnote for more explanation);
 :grey_question: Unknown (requires an issue).
 
 <sup>1</sup> See issue https://github.com/Rensvandeschoot/software-overview-machine-learning-for-screening-text/issues/34
+<sup>2</sup> Currently only supports relevant records being uploaded as training data. The model will start getting built once the user has screened at least one included and one excluded reference.
```
Comment (gimoAI) [inline README.md:130]: See comment above, we might want to change this footnote and explain that. `The model will start getting built once the user has screened at least one included and one excluded reference. ` may be incorrect

File: README.md [hunk 4]
```diff
@@ -139,16 +140,16 @@ The tables below provide an overview of the machine learning properties.
...
+|       [Colandr](#colandr)                     |        :x:         |        :x:         |        :x:         |    A     |     :white_check_mark:      |  :x: (10)  |        :x:        |
+| [EPPI-Reviewer](#eppi-reviewer)               |        :x:         |        :x:         |        :x:         |    M     |     :white_check_mark:      |    :x:     |        :x:        |
+|      [FASTREAD](#fastread)                    |        :x:         |        :x:         |        :x:         |    M     |             :x:             |    :x:     |        :x:        |
+|        [Rayyan](#rayyan)                      |        :x:         |        :x:         |        :x:         |    M     |     :white_check_mark:      |    :x:     |        :x:        |
+|  [RobotAnalyst](#robotanalyst)                |        :x:         |        :x:         |        :x:         |    M     | :grey_question:<sup>2</sup> |    :x:     |        :x:        |
+| [SWIFT-Active Screener](#swift-activescreener)|        :x:         |        :x:         |        :x:         |    A     |             :x:             |    :x:     |        :x:        |
```
Comment (gimoAI) [inline README.md:152]: - I couldn't find whether you can `Continue` when a new model is being built: so question mark there with a footnote?
- Batch size should be 30

File: README.md [hunk 5]
```diff
@@ -211,7 +212,7 @@ The software will recognize all previous labeling decisions, and a new model can
 |      [FASTREAD](#fastread)      |                TF-IDF                |               SVM                |          S, A, W, M          |      C, U      |
 |        [Rayyan](#rayyan)        |     :grey_question:<sup>4</sup>      |               SVM                | :grey_question:<sup>4</sup>  |      C, U      |
 |  [RobotAnalyst](#robotanalyst)  |        TF-IDF + BOW + LDA2vec        |               SVM                | :grey_question:<sup>5</sup>  |  R, C, U, Cl   |
-|  [SWIFT-Review](#swift-review)  |             TF-IDF + LDA             |                LL                | S:grey_question:<sup>6</sup> |    C, U, Cl    |  
+|  [SWIFT-Active Screener](#swift-activescreener)  |             TF-IDF + LDA             |                LL                | S:grey_question:<sup>6</sup> |    C, U, Cl    |
```
Comment (gimoAI) [inline README.md:215]: - Feature extractor: TF-IDF (no LDA)
- In the paper they mention that query strategy is only certainty based

**Review by gimoAI — COMMENTED** [review]
I went over the specs of SWIFT-active screener, using mainly these sources:
- [FAQ](https://www.sciome.com/swift-activescreener/knowledgebase/)
- [Paper](https://doi.org/10.1016/j.envint.2020.105623)

File: README.md (hunk 1 above)
Comment (Rensvandeschoot) [reply README.md:79]: solved in https://github.com/Rensvandeschoot/software-overview-machine-learning-for-screening-text/pull/53

File: README.md (hunk 2 above)
Comment (Rensvandeschoot) [reply README.md:122]: solved in https://github.com/Rensvandeschoot/software-overview-machine-learning-for-screening-text/pull/53

File: README.md (hunk 4 above)
Comment (Rensvandeschoot) [reply README.md:152]: solved in https://github.com/Rensvandeschoot/software-overview-machine-learning-for-screening-text/pull/53

File: README.md (hunk 5 above)
Comment (Rensvandeschoot) [reply README.md:215]: solved in https://github.com/Rensvandeschoot/software-overview-machine-learning-for-screening-text/pull/53

File: README.md (hunk 3 above)
Comment (Rensvandeschoot) [reply README.md:130]: solved in https://github.com/Rensvandeschoot/software-overview-machine-learning-for-screening-text/pull/53
